templates/Default/admin/Default/enable_game.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// This var is passed by the processing file if we enabled a game
+if (!empty($ProcessingMsg)) {
+	echo $ProcessingMsg;
+}
+
+if (empty($DisabledGames)) {
+	echo "<p>All games are already enabled!</p>";
```

Comment:
Can you do this via
```
?><p>All games are already enabled!</p><?php
```
instead please, it lets the html easily be syntax highlighted/validated by editors (you can see github has highlighted the html below due to being outside of the php tags)